_data/simple-icons.json

```
@@ -3375,6 +3375,11 @@
             "hex": "4695EB",
             "source": "https://design.jboss.org/quarkus/"
         },
+        {
+            "title": "Quest",
+            "hex": "FB4F14",
+            "source": "https://en.wikipedia.org/wiki/Quest_Software"
```

I think (https://www.quest.com/legal/trademark-information.aspx) works better as source here. The brand guidelines are available from this page and the logo svg is embedded in same .pdf.
```suggestion
            "source": "https://www.quest.com/legal/trademark-information.aspx"
```

---

:+1: Done